src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
```

Comment:
Ensure that each BEM block has its own separate file as per the task requirements. This helps in maintaining modularity and reusability of components.